README.md

```diff
@@ -1,6 +1,29 @@
 <p align="center">
   <img src="java-core/src/main/java/ai/certifai/maven/metadata/java.png">
-</p> 
+</p>
+
+<p>
+  <p align="center">
+    <a href="https://sonarcloud.io/dashboard?id=CertifaiAI_learn-java-the-certifai-way">
+        <img alt="Sonar Cloud" src="https://sonarcloud.io/images/project_badges/sonarcloud-white.svg">
+    </a>
+</p>
+
+<p>
+  <p align="center">
+    <a href="https://github.com/CertifaiAI/learn-java-the-certifai-way/blob/main/LICENSE">
+        <img alt="GitHub" src="https://img.shields.io/github/license/CertifaiAI/TrainingLabs.svg">
```

Comment:
https://img.shields.io/github/license/CertifaiAI/learn-java-the-certifai-way.svg